### PR TITLE
Initial saps dispatcher project

### PR DESCRIPTION
**Pull request by thiagoyeds**

### CODE REVIEW

File: deploy/config/archiver.conf [hunk 1]
```diff
@@ -0,0 +1,58 @@
+##### Catalog Service #####
```
Comment (raonismaneoto) [inline deploy/config/archiver.conf]: is this archiver.conf needed by the dispatcher?

Comment (raonismaneoto) [reply deploy/config/archiver.conf]: same for the others conf files

File: docker/dockerfiles/archiver [hunk 2]
```diff
@@ -0,0 +1,29 @@
+FROM maven:3.6.3-jdk-8 AS builder
```
Comment (raonismaneoto) [inline docker/dockerfiles/archiver]: archiver dockerfile shouldn't be here

File: docs/end-to-end-test.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# End to end test using NOP algorithms
```
Comment (raonismaneoto) [inline docs/end-to-end-test.md]: it should be in the saps-quality-assurance repo, right?

**Review by raonismaneoto — CHANGES_REQUESTED** [review]
I've made some comments. Pay attention in the files you're including in the repo. Only things related to dispatcher should be here. Do the same for the other repos, please.

File: docs/end-to-end-test.md (hunk 3 above)
Comment (thiagoyeds) [reply docs/end-to-end-test.md]: done

File: deploy/config/archiver.conf (hunk 1 above)
Comment (thiagoyeds) [reply deploy/config/archiver.conf]: done

File: docker/dockerfiles/archiver (hunk 2 above)
Comment (thiagoyeds) [reply docker/dockerfiles/archiver]: done

**Review by raonismaneoto — APPROVED** [review]

**Pull request merged**